pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -28,12 +31,57 @@ buildPythonPackage rec {
     numpy
     scipy
     h5py
+    cppe
   ];
 
+  checkInputs = [ nose nose-exclude ];
+
   PYSCF_INC_DIR="${libcint}:${libxc}:${xcfun}";
```

Comment:
```suggestion
  PYSCF_INC_DIR="${libcint}:${libxc}:${xcfun}";
```
I think this should be a ``preBuild`` export?

Reply:
Already in the `preConfigure` but yes :)